src/diagnostics/diagnostic-codes.md

```diff
@@ -1,5 +1,5 @@
 # Diagnostic Codes
-We generally try assign each error message a unique code like `E0123`. These
+We generally try to assign each error messages a unique code like `E0123`. These
```

Comment:
This should still be 'message': 'each message' refers to a single one.

Reply:
Thanks for your review. I fixed it.